user/languages/r.md

```diff
@@ -188,6 +188,25 @@ r: bioc-release
 Travis CI will use the proper R version for that version of Bioconductor and
 configure Bioconductor appropriately for installing dependencies.
 
+### Packrat
+
+If you want Travis CI to use your project-specific [packrat](https://rstudio.github.io/packrat/) package library, rather than the default behaviour of downloading your package dependencies from CRAN, you can add this to your `.travis.yml`:
+
+```yaml
+install:
+  - R -e "0" --args --bootstrap-packrat
```

Comment:
I haven't had much luck with this, when I remove `-e "0"` the build stalls and quits: https://travis-ci.org/benmarwick/mjbtramp/builds/158881647, but works when I put it back in: https://travis-ci.org/benmarwick/mjbtramp/builds/158882619